packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -83,7 +84,14 @@ class TestCommand extends FastFlutterCommand {
         abbr: 'j',
         defaultsTo: math.max<int>(1, platform.numberOfProcessors - 2).toString(),
         help: 'The number of concurrent test processes to run.',
-        valueHelp: 'jobs');
+        valueHelp: 'jobs'
+      )
+      ..addFlag('no-assets',
```

Comment:
I would name this something more specific like `--no-test-assets`.

Reply:
fixed